packages/web/src/components/user-badges/FollowsYouBadge.tsx

```diff
@@ -8,7 +8,11 @@ const messages = {
   followsYou: 'Follows You'
 }
 
-export default function FollowsYouBadge() {
+const FollowsYouBadge = ({ className = '' }: { className?: string }) => {
```

Comment:
i think doesn't need default prop technically, i guess unless wm is different from 'classnames'?